src/components/BookList.jsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please avoid disabling linter rules as they serve as a way of having a consistent code format and ensuring best practices! 🤓 🔧